controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -516,6 +549,9 @@ func (r *StorageConsumerReconciler) reconcileCephFilesystemSubVolumeGroup(
 		}
 		svg.Spec.FilesystemName = cephFs.Name
 		svg.Spec.Name = subVolumeGroupName
+		if forceDelete {
+			util.AddAnnotation(svg, util.RookForceDeletionAnnotationKey, strconv.FormatBool(true))
+		}
```

Comment:
Please remove this directly below the owner ref because this is metadata and not spec